templates/Search/form.html.twig

```diff
@@ -1,69 +1,104 @@
-        </div>
-        <div class="col-md-2">
-            <select class="input-group form-control" name="field" id="field">
-                    <option
-                            {% if field is empty %}
-                                selected
-                            {% endif %}
-                            value>
-                        -- All --
-                    </option>
-                    <option
-                            value={{ constant('App\\Util\\Search::ELASTIC_INDEX_MAPPING_TITLE') }}
-                            {% if field is defined and field is not empty and field == constant('App\\Util\\Search::ELASTIC_INDEX_MAPPING_TITLE') %}
-                            selected
-                            {% endif %}>
-                        Title
-                    </option>
-                    <option
-                            value={{ constant('App\\Util\\Search::ELASTIC_INDEX_MAPPING_ABSTRACT') }}
-                             {% if field is defined and field is not empty and field == constant('App\\Util\\Search::ELASTIC_INDEX_MAPPING_ABSTRACT')%}
-                            selected
-                            {% endif %}>
-                        Abstract
-                    </option>
-                    <option
-                            value={{ constant('App\\Util\\Search::ELASTIC_INDEX_MAPPING_AUTHORS') }}
-                             {% if field is defined and field is not empty and field == constant('App\\Util\\Search::ELASTIC_INDEX_MAPPING_AUTHORS')%}
-                            selected
-                            {% endif %}>
-                        Author
-                    </option>
-                    <option
-                            value={{ constant('App\\Util\\Search::ELASTIC_INDEX_MAPPING_THEME_KEYWORDS') }}
-                             {% if field is defined and field is not empty and field == constant('App\\Util\\Search::ELASTIC_INDEX_MAPPING_THEME_KEYWORDS')%}
-                            selected
-                            {% endif %}>
-                        Theme Keywords
-                    </option>
-            </select>
-        </div>
-        <div class="input-group col-md-4">
-            <input type="search" id="searchBox" name="query" class="search-query form-control" placeholder="Search..."
-                   value = "{{ query|default }}">
-            {% if page is defined and page is not null%}
-                <input type="hidden" id="pageNo" name="page" value="{{ page | default }}">
-            {% endif %}
-            <span class="input-group-btn">
-                <button id="searchSubmit" class="btn btn-danger" type="submit">
-                    <span class=" glyphicon glyphicon-search"></span>
-                </button>
-            </span>
+<!-- ========================= SECTION CONTENT ========================= -->
+<section class="section-content bg pt-5">
+    <div class="container">
+        <div class="card card-header">
```

Comment:
Card may not be the right thing to use here. If you just want a different background there are other ways to do it. Also the card doesn't scale right it seems.